subsys/bluetooth/mesh/rpr_cli.c

```diff
@@ -131,7 +131,6 @@ static int handle_link_report(struct bt_mesh_model *mod, struct bt_mesh_msg_ctx
 	struct bt_mesh_rpr_cli *cli = mod->user_data;
 	struct bt_mesh_rpr_link link;
 	uint8_t reason = PROV_ERR_NONE;
```

Comment:
Little bit on the side: What is the purpose of reason here other than having something to pull the value into? Could that be removed and return if buf->len was greater than 1.

Reply:
True, I can add it to log message.

Reply:
Done